main.go

```diff
@@ -33,12 +19,13 @@ func RetrieveSecret(variableName string) {
 		printAndExit(err)
 	}
 
-	fmt.Print(value)
+	os.Stdout.Write([]byte(value))
+
 }
 
 func main() {
 	if len(os.Args) != 2 {
-		fmt.Println("A variable name must be given as the first and only argument!")
+		os.Stderr.Write([]byte("A variable name must be given as the first and only argument!"))
```

Comment:
You missed the endline character here.

Additionally you're casting a string to binary; I'm not sure what `go` would do here encoding-wise, but it smells wrong to me and can possibly break on the one very popular platform that doesn't use UTF-8 like every other OS on the planet. I think you should use `WriteString` instead -- since you are indeed writing a string, not binary data. 

Or, ideally, `fmt.Fprintln`, so the code is exactly equivalent save different stream. I suspect you just missed that method, golang docs are sometimes hard to grep :)